cmd/blockchaincmd/create.go

```diff
@@ -225,24 +232,27 @@ func createBlockchainConfig(cmd *cobra.Command, args []string) error {
 
 	sc := &models.Sidecar{}
 
-	if err = promptValidatorManagementType(app, sc); err != nil {
-		return err
-	}
-
-	if !sc.PoA() && createFlags.poaValidatorManagerOwner != "" {
-		return errors.New("--poa-manager-owner flag cannot be used when blockchain validator management type is not Proof of Authority")
+	if sovereign {
+		if err = promptValidatorManagementType(app, sc); err != nil {
+			return err
+		}
+		if !sc.PoA() && createFlags.poaValidatorManagerOwner != "" {
+			return errors.New("--poa-manager-owner flag cannot be used when blockchain validator management type is not Proof of Authority")
+		}
 	}
 
 	if vmType == models.SubnetEvm {
-		if sc.PoA() {
-			if createFlags.poaValidatorManagerOwner == "" {
-				createFlags.poaValidatorManagerOwner, err = getValidatorContractManagerAddr()
-				if err != nil {
-					return err
+		if sovereign {
```

Comment:
do I understand this correctly that it can be replaced with `if sovereign && sc.PoA()`